go-controller/pkg/ovn/egressgw.go

```diff
@@ -349,7 +349,13 @@ func (oc *Controller) addHybridRoutePolicyForPod(podIP net.IP, node string) erro
 		if err != nil {
 			return fmt.Errorf("failed to parse gateway router join interface IP: %s, err: %v", grJoinIP, err)
 		}
+		var matchDst string
+		for _, clusterSubnet := range config.Default.ClusterSubnets {
+			matchDst += fmt.Sprintf(" && %s.dst != %s", l3Prefix, clusterSubnet.CIDR)
```

Comment:
as with the other PR this needs to check `utilnet.isIPv6CIDR(clusterSubnet.CIDR)` to handle dual-stack (and again below when deleting)

Reply:
will fix